File: C/beginner/1065.cpp
#include <stdio.h>
 
int main() {
	float num[5];
	int count = 0;
	
    for(int i = 0; i < 5; i++){
    	scanf("%f", &num[i]);
    	if((int)num[i] % 2 == 0) {
			count++;
		}
    }
    
    printf("%d valores pares\n", count);
    
    
    return 0;
}

File: C#/begginer/1065.cs
using System; 

class URI {

  static void Main(string[] args) { 
    int count = 0;
    for(int i = 0; i < 5; i++) {
      int input = int.Parse(Console.ReadLine());
      if (input % 2 == 0) {
        count++;
      }
    }
    Console.WriteLine($"{count} valores pares");
  }

}
